src/Meshes/buffer.ts

```diff
@@ -261,6 +261,7 @@ export class VertexBuffer {
         } else {
             this._instanced = true;
         }
+        this._computeHashCode();

```

Comment:
Is this also not heavy for gl ?

Reply:
The computation is only doing some bit operations and is done once per vertex buffer, so I don't think it's too much of a problem (we don't create a buffer each frame).

I have moved this computation here because else it was (re)done each time we access the render pipeline cache in WebGPU, so once per draw call.